paasta_tools/utils.py

```diff
@@ -3415,6 +3419,18 @@ def format_tag(tag: str) -> str:
     return "refs/tags/%s" % tag
 
 
+def build_image_identifier(
+    git_sha: str, sha_len: Optional[int] = None, image_version: Optional[str] = None
+) -> str:
```

Comment:
Seems useful! Which places were you planning on using this w/ a limited sha?

Reply:
I noticed that in places like [this](self.commit[:8]), `self.commit[:8]` is used to identify particular images, so was planning on using the limited sha there